src/context.ts

```diff
@@ -27,8 +30,20 @@ function getUserId(token?: string): string {
       throw new ApolloError(err.name === 'TokenExpiredError' ? 'Token expired' : 'Invalid authentication token');
     }
     return decoded;
-  }) as unknown as { userId: string };
-  return verificationTokenResult.userId;
+  }) as unknown as { sessionId: string };
```

Comment:
Do you also want to move `verify` to the newly created `token.ts` under `validateAndParse` and properly type it there?

Reply:
Also, I thought `ApolloError`s were replaced with graphql errors in a previous PR, but I still can see one above. Please take care of it